### PR TITLE
Add Vaibhavi24 as Mentor for C++

### PR DESCRIPTION
**Pull request by Vaibhavi24**

### CODE REVIEW

**Comment by Vaibhavi24** [comment]
@tehsphinx Pls review :)

File: mentors/cpp/v.json [hunk 1]
```diff
@@ -1 +1,10 @@
-[]
+[
+	{
+	"github_username": "Vaibhavi24",
+    "name": "Vaibhavi Singh",
+    "link_text": "Linkedin profile",
+    "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
+    "avatar_url": null,
+    "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
+    }
```
Comment (rpottsoh) [inline mentors/cpp/v.json:9]:
```suggestion
   {
      "github_username": "Vaibhavi24",
      "name": "Vaibhavi Singh",
      "link_text": "Linkedin profile",
      "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
      "avatar_url": null,
      "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
   }
```

Just some minor indentation corrections to make.

Comment (Vaibhavi24) [reply mentors/cpp/v.json]: Thanks @rpottsoh! Made the changes! Pls review :)

File: mentors/cpp/v.json [hunk 2]
```diff
@@ -1 +1,10 @@
-[]
+[
+	{
+        "github_username": "Vaibhavi24",
+        "name": "Vaibhavi Singh",
+        "link_text": "Linkedin profile",
+        "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
+        "avatar_url": null,
+        "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
+    }
```
Comment (rpottsoh) [inline mentors/cpp/v.json:9]:
```suggestion
  {
    "github_username": "Vaibhavi24",
    "name": "Vaibhavi Singh",
    "link_text": "Linkedin profile",
    "link_url": "https://www.linkedin.com/in/vaibhavi-singh-45619a138/",
    "avatar_url": null,
    "bio": "I've written code in C++ for more than 6 yrs. Mentored in Open Source programs, love to understand and simplify abstract concepts."
  }
```
It should look like this.  Feel free to just commit this suggestion.

**Pull request merged**